src/storage/btree_load.h

```diff
@@ -180,7 +180,8 @@ extern int btree_get_next_overflow_vpid (PAGE_PTR page_ptr, VPID * vpid);
       if (!((p_mvcc_rec_header)->mvcc_flag & OR_MVCC_FLAG_VALID_DELID)) \
         { \
           (p_mvcc_rec_header)->mvcc_flag |= OR_MVCC_FLAG_VALID_DELID; \
-          (p_mvcc_rec_header)->delid_chn.mvcc_del_id = MVCCID_NULL; \
+          (p_mvcc_rec_header)->mvcc_del_id = MVCCID_NULL; \
+          (p_mvcc_rec_header)->chn = 0; \
```

Comment:
The chn is not important in BTREE header. Even though it uses the same structure, it is supposed to use only ins_id and del_id.